### PR TITLE
[BB-05] Responsive website

### PR DESCRIPTION
**Pull request by KreatorOf**
Context:  background-color doesn't fit screen when resize on my side.

Youtrack: [BB-05](https://irisrobotics.youtrack.cloud/issue/PMI-34)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -3,84 +3,77 @@
...
-	font-size: 100px;
-	margin-top: 20px;
+	font-size: 120px; /* Increase font size */
+	color: white;
 }
 
 .slogan {
-	margin-top: -87px;
-	margin-bottom: 37px;
+	margin-top: -100px; /* Adjust margin for more space */
```
Comment (PhantHive) [inline src/App.css:33]: Hmmm, are you sure this was needed?

**Review by PhantHive — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (PhantHive) [reply src/App.css:33]: I think we could avoid that along the CSS by using more flex box

**Comment by KreatorOf** [comment]
I add an AI response field, check the code and we can merge on main and delete this branch ! ;)

YouTrack: [BB-06](https://irisrobotics.youtrack.cloud/issue/PMI-39)

**Comment by PhantHive** [comment]
We cannot merge this branch until it's fully responsive :|

File: src/App.css [hunk 2]
```diff
@@ -101,10 +110,9 @@
...
-	align-items: center;
-	justify-content: center;
-	width: 50%;
-	margin: 20px;
+.footer {
+	grid-column: 1 / span 4;
+	font-style: italic;
+	font-size: 10px; /* Increase font size */
+	color: white;
 }
```
Comment (PhantHive) [inline src/App.css:118]: Add a media section for phones

```css
@media screen and (max-width: 800px) {
	.title-slogan {
		display: grid;
		grid-column: 1 / span 4;
		align-items: center;
		justify-content: center;
		gap: 20px;
	}
	.bb-title {
		color: #FFF;
		font-size: 55px; 
		font-style: normal;
		line-height: normal;
		text-align: center;
	}
	.slogan {
		color: #C6C8DF;
		font-size: 23px; 
		font-style: normal;
		line-height: normal;
		text-align: center;
	}
}
```

**Review by PhantHive — CHANGES_REQUESTED** [review]
Add the  media section  and adapt it to fits the need of the mobile version.

File: src/components/output.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+export function Outputs({ submittedRequest }: { submittedRequest: string }) {
```
Comment (PhantHive) [inline src/components/output.tsx:1]: Should be const arrow function ☀️🥹

**Review by PhantHive — APPROVED** [review]

**Review by PhantHive — APPROVED** [review]
Approved.

**Pull request merged**